src/services/FaqServices.ts

```diff
@@ -14,13 +14,14 @@ export class FaqServices implements IFaqServices {
   constructor(serviceScope: ServiceScope) {
     serviceScope.whenFinished(() => {
       this._spHttpClient = serviceScope.consume(SPHttpClient.serviceKey);
+      this._pageContext = serviceScope.consume(PageContext.serviceKey);
       this._currentWebUrl = this._pageContext.web.absoluteUrl;
-      // this._currentWebUrl = 'https://devgcx.sharepoint.com/sites/Support';
+      this._currentWebUrl = 'https://devgcx.sharepoint.com/sites/Support';
```

Comment:
Is currentWebUrl duplicate?

Reply:
I think I hardcoded for testing in workbench. Sorry I will change it back.